src/components/scripts/google-analytics.tsx

```diff
@@ -1,4 +1,5 @@
 export function GoogleAnalytics() {
+  if (process.env.GA_TRACKING_ID === undefined) return null
```

Comment:
Since we add and remove this line pretty much in every other commit: Maybe remove the conditional from `_document` and always render `GoogleAnalytics` since you added the conditional here.

Reply:
Are `process.env.GA_TRACKING_ID` and `process.env.NEXT_PUBLIC_GA_TRACKING_ID` identical?

Reply:
Ah, this should be `process.env.NEXT_PUBLIC_GA_TRACKING_ID ` everywhere

Reply:
:D

Reply:
My bad 😓

Reply:
No, no, just celebrating the good catch :)